doc/source/notebooks/advanced/variational_fourier_features.ipynb

```diff
@@ -4,11 +4,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Variational Fourier Features in the GPflow framework\n",
+    "# Variational Fourier features in the GPflow framework\n",
     "\n",
     "In this notebook we demonstrate how new types of inducing variables can easily be incorporated in the GPflow framework. As an example case, we use the variational Fourier features from [Hensman, Durrande, and Solin (JMLR 2018)](http://jmlr.csail.mit.edu/papers/v18/16-579). All equation and table references are to this paper.\n",
     "\n",
-    "**Note:** we cannot yet use Fourier features within the multi-output framework, as `Kuu` and `Kuf` for SharedIndependent and SeparateIndependent inducing variables assume that the sub-inducing variable's covariances are simply computed as dense Tensors. Moreover, the `conditional` is not able to make use of the structure in `Kuu` and `Kuf` as it has to dispatch on the *arguments* to `Kuu` and `Kuf` instead..."
+    "**NOTE:** We cannot yet use Fourier features within the multioutput framework, because `Kuu` and `Kuf` for `SharedIndependent` and `SeparateIndependent` inducing variables assume that the sub-inducing variable's covariances are simply computed as dense Tensors. Moreover, the `conditional` cannot make use of the structure in `Kuu` and `Kuf` because it has to dispatch on the *arguments* to `Kuu` and `Kuf` instead."
```

Comment:
```suggestion
    "**NOTE:** We cannot yet use Fourier features within the multi-output framework, because `Kuu` and `Kuf` for `SharedIndependent` and `SeparateIndependent` inducing variables assume that the sub-inducing variable's covariances are simply computed as dense Tensors. Moreover, the `conditional` cannot make use of the structure in `Kuu` and `Kuf` because it has to dispatch on the *arguments* to `Kuu` and `Kuf` instead."
```